src/App.js

```diff
@@ -1,38 +1,11 @@
 import './index.css';
+import { Calculate } from './compnents/Calculator';
 
 function App() {
   return (
-    <>
-      <form>
-        <div><input type="text" /></div>
-        <div className="buttons">
```

Comment:
- [ ] kindly consider making sure that this div displays the inputs as intended  to make it function as it is required while doing so consider making sure that  it  responds to all user interactions